NOTICE.txt

```diff
@@ -1,55 +1,30 @@
 
-Copyright © 2023 MarkLogic Corporation.  MarkLogic and MarkLogic logo are trademarks or registered trademarks of MarkLogic Corporation in the United States and other countries.  All other trademarks are the property of their respective owners.
- 
-This project is licensed under the Apache License, Version 2.0 (the "License"); you may not use this project except in compliance with the License. You may obtain a copy of the License at
+Copyright © 2024 MarkLogic Corporation. All Rights Reserved.
 
-http://www.apache.org/licenses/LICENSE-2.0 
+To the extent required by the applicable open-source license, a complete machine-readable copy of the source code corresponding to such code is available upon request. This offer is valid to anyone in receipt of this information and shall expire three years following the date of the final distribution of this product version by Progress Software Corporation. To obtain such source code, send an email to Legal-thirdpartyreview@progress.com. Please specify the product and version for which you are requesting source code.
 
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+Third Party Notices
 
-To the extent required by the applicable open-source license, a complete machine-readable copy of the source code corresponding to such code is available upon request. This offer is valid to anyone in receipt of this information and shall expire three years following the date of the final distribution of this product version by MarkLogic Corporation. To obtain such source code, send an email to Legal-thirdpartyreview@progress.com. Please specify the product and version for which you are requesting source code.
+MarkLogic Data Hub Framework 6.1.1 (Apache-2.0)
+ml-app-deployer 5.0.0 (Apache-2.0)
+Jackson Dataformat 2.14.1 (Apache-2.0)
 
-The following software may be included in this project (last updated October 3, 2023):
+Common Licenses
 
--------------------------------------------------------------------------
-Third Party Components
+Apache License 2.0 (Apache-2.0)
 
-Jackson Annotations 2.14.1
-Attribution Statements
-http://wiki.fasterxml.com/JacksonHome
-
-Copyright Statements
-Copyright ©2009 FasterXML, LLC
-
-License Text (http://spdx.org/licenses/Apache-2.0)
-Made available under the Apache License 2.0. See Appendix for full text.
+Third-Party Components
 
-Source materials are available for download at: http://github.com/FasterXML/jackson
+The following is a list of the third-party components used by MarkLogic® Connector for Kafka v1.10.0 (last updated November 1, 2024):
 
-Jackson Core 2.14.1
-Attribution Statements
-http://wiki.fasterxml.com/JacksonHome
+MarkLogic Data Hub Framework 6.1.1 (Apache-2.0)
+https://developer.marklogic.com/products/data-hub/
```

Comment:
Use the Maven Central URL - e.g. https://repo1.maven.org/maven2/com/marklogic/marklogic-data-hub/6.1.1/ . Use the artifact name as well. Same goes for ml-app-deployer below.